browser-sign-in/ios/browserSignInTests/browserSignInTests.m

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+
+#import <React/RCTLog.h>
+#import <React/RCTRootView.h>
+
+#define TIMEOUT_SECONDS 600
+#define TEXT_TO_LOOK_FOR @"Welcome to React Native!"
+
+@interface browserSignInTests : XCTestCase
```

Comment:
browserSignInTests -> BrowserSignInTests

Reply:
Deleted this test package along with tvOS since it isn't needed